addons/l10n_jo_edi/models/account_edi_xml_ubl_21_jo.py

```diff
@@ -409,3 +410,16 @@ def _export_invoice_vals(self, invoice):
         self._aggregate_totals(vals['vals'])
 
         return vals
+
+    def _export_invoice(self, invoice):
+        # _export_invoice normally cleans up the xml to remove empty nodes.
```

Comment:
You forgot the:
`# EXTENDS account.edi.xml.ubl_21`

Reply:
I didn't find it necessary, as it is never used anywhere else in the file, and the model inherits `account.edi.xml.ubl_21` directly.